grid/recruitment/jobs/maps/workable.suma

```diff
@@ -0,0 +1,155 @@
+profile = "recruitment/jobs@1.0"
+provider = "workable"
+
+map ListJobs {
+    http GET "/spi/v3/jobs" {
+      security "bearer_token"
+
+      request {
+         query {
+          state = input.state
+          limit = input.limit
+          since_id = input.page
+          include_fields = 'requirements,description,benefits'
+        }
+
+        headers {
+          'X-WORKABLE-CLIENT-ID' = parameters.CLIENT_ID
+        }
+      }
+
+      response 200 "application/json" {
+          jobs = body.jobs.map(job => {
+            return {
+              id: job.shortcode,
+              name: job.title,
+              description: job.description,
+              requirements: [job.requirements],
+              benefits: [job.benefits],
```

Comment:
question: don't we want to parse the html in case it contains list of requirements/benefits?

Reply:
I do not think we able to parse the html reliably. Html of requirements is created in rich text editor it can contain basically any html content. I would suggest to support two different benefits and requirements types: string | [string] to make it clear what kind of benefit content is provided.

Reply:
Ok, agree